user-guide/Advanced_Functionality/Swarming/SwarmingFAQ.md

```diff
@@ -0,0 +1,72 @@
+---
+uid: SwarmingFAQ
+---
+
+# Frequently Asked Questions
+
+## Can I Swarm X/Y/Z?
+
+Swarming will gradually become available for more and more functionality in DataMiner.
+As of DataMiner 10.5.1, only basic elements are supported.
```

Comment:
Link to https://docs.dataminer.services/user-guide/Advanced_Functionality/Swarming/Swarming.html#upcoming-features instead?

Reply:
Done